test/e2e-ocl/onclusterlayering_test.go

```diff
@@ -207,20 +211,36 @@ func TestMissingImageIsRebuilt(t *testing.T) {
 	t.Logf("MachineOSBuild %q has completed and produced image: %s", secondFinishedBuild.Name, secondImagePullspec)
 	waitForMOSCToGetNewPullspec(ctx, t, cs, moscName, secondImagePullspec)
 
-	// Delete the first image
+	// Delete the first image- simulating image deletion
 	t.Logf("Deleting image %q", firstImagePullspec)
 	istName := fmt.Sprintf("os-image:%s", firstMOSB.Name)
 	err = cs.ImageStreamTags(ctrlcommon.MCONamespace).Delete(ctx, istName, metav1.DeleteOptions{})
 	require.NoError(t, err)
 	kubeassert.ImageDoesNotExist(istName)
 	t.Logf("Deleted image %q", firstImagePullspec)
-	// Delete the MC
+
+	// Delete the USBGuard image (second build) by digest so reuse will fail
+	t.Logf("Deleting usbguard IST for MOSB %q", secondMOSB.Name)
```

Comment:
we should only be deleting the image from the first build in the test, any reason why we are doing that in addition to deleting the usbguard image?